articles/search/search-howto-managed-identities-cosmos-db.md

```diff
@@ -22,10 +22,17 @@ You can use a system-assigned managed identity or a user-assigned managed identi
 
 * [Create a managed identity](search-howto-managed-identities-data-sources.md) for your search service.
 
-* [Assign a role](search-howto-managed-identities-data-sources.md#assign-a-role) in Azure Cosmos DB.
-
+* [Control plane role assignment](search-howto-managed-identities-data-sources.md#assign-a-role) in Azure Cosmos DB.
   For data reader access, you'll need the **Cosmos DB Account Reader** role and the identity used to make the request. This role works for all Azure Cosmos DB APIs supported by Cognitive Search. This is a control plane RBAC role. 
 
+* Data Plane Role assignment: Follow this https://learn.microsoft.com/en-us/azure/cosmos-db/how-to-setup-rbac 
+to know more about data plane role assignments for CosmosDB
+
+* *Only applicable for Cosmos Sql ApiKind*:
```

Comment:
For lines 31-33, I would follow Cosmos DB conventions and go with: 

For Cosmos DB for NoSQL, you can optionally [enforce role-based access control as the sole authentication method](/azure/cosmos-db/how-to-setup-rbac#disable-local-auth) for data connections by setting  `disableLocalAuth` to `true` for your Cosmos DB account.

Reply:
This makes sense, will add this.

Reply:
@vaishalishah11 as per our offline conversation, as long as Heidi's verbatim is used to reference this before signing off, I approve the sign off right after. Everything else looks good to me. Thanks!